website/write_events.php

```diff
@@ -101,4 +101,32 @@ function write_events($date)
 		print("			</tr>\n");
 		print("		</table>\n");
 	}
+
+function write_ch_events($date)
+	{
+		include ("globals.php");
+		
+		//	start the display
+		print("<table width=780>\n");
+		print("	<tr>\n");
+		print("		<td align=\"left\"><font size=\"-1\">\n");
+
+		$file = "${arm_data_path}data/" . $dirdate . "/meta/arm_ar_summary_issue_time_" . $date . ".txt";	
+		
+		if (file_exists($file))
+		{
+			$times = file($file);
+			$time1 = $times[0];
+			$time2 = $times[1];
+		}
+		else
+		{	
+			$time1 = "some time";
+			$time2 = "some time";
+		}

```

Comment:
What are these `time`s for? You don't want that to be shown in the page, do you?